<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>893</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>893</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: rgb(50, 50, 50);
}</string>
  </property>
  <widget class="QLabel" name="label_sum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>200</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 25px;
color: white;</string>
   </property>
   <property name="text">
    <string>Сумма кредита, руб</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_month">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>200</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 25px;
color: white;</string>
   </property>
   <property name="text">
    <string>Срок, мес</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_proc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>200</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 25px;
color: white;</string>
   </property>
   <property name="text">
    <string>Процентная ставка, %</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>201</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 25px;
color: white;</string>
   </property>
   <property name="text">
    <string>Тип ежемесячных платежей</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_monthToPay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 25px;
color: rgb(182, 56, 56);</string>
   </property>
   <property name="text">
    <string>Ежемесячный платеж, руб</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_procToPay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 25px;
color: rgb(182, 56, 56);</string>
   </property>
   <property name="text">
    <string>Начисленные проценты, руб</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sumToPay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 25px;
color: rgb(182, 56, 56);</string>
   </property>
   <property name="text">
    <string>Общий долг, руб</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_monthToPay_B">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>430</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(182, 56, 56);;
border-radius: 25px;
color: white;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>591</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(75, 75, 75);
	color: white;
	border: 1px solid gray;
	border-radius: 45px;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
									stop: 0 #dadbde, stop: 1 #D3D3D3);
	color: black;
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>490</y>
     <width>261</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(211, 211, 211);
	color: rgb(182, 56, 56);
	border: 1px solid black;
	border-radius: 45px;
	border-style: outset;
}
QPushButton:pressed {
	background-color: rgb(182, 56, 56);
	color: rgb(211, 211, 211);
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_period">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>120</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	color: white;
	border-radius: 25px;
	border: 1px solid gray;
}</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_sumToPay_B">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(182, 56, 56);;
border-radius: 25px;
color: white;
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_procToPay_B">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(182, 56, 56);;
border-radius: 25px;
color: white;
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_type1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_type2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Диф-ный</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBo_sumCredit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>120</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	color: white;
	border-radius: 25px;
	border: 1px solid gray;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>9223372036854775808.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_procent">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>120</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	color: white;
	border-radius: 25px;
	border: 1px solid gray;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>521</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
